packages/@rnw-scripts/doxysaurus/src/docusaurus.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+//
+// Copy generated documentation files to Docusaurus location.
+//
+
+import path from 'path';
+import {log} from './logger';
+import {promises as fs} from 'fs';
+
+export async function copyDocusaurusFiles(files: string[]) {
+  const docsPath = process.env.DOCUSAURUS_DOCS;
```

Comment:
This seems like it would be better as a CLI arg instead of env variable. CLI help can guide if required, check if present early, make changing path on invocation easier, etc.

Reply:
The code is changed in commit https://github.com/microsoft/react-native-windows/pull/7044/commits/9d9217ebec01386c2c58eec42c8bed6c73f76218 to use the new `output` CLI parameter.